xarray/backends/common.py

```diff
@@ -8,7 +8,7 @@
 
 from ..conventions import cf_encoder
 from ..core import indexing
-from ..core.pycompat import dask_array_type
+from ..core.dask_array_compat import is_duck_dask_array
```

Comment:
let's keep this in pycompat as earlier. `dask_array_compat` contains backports of newer dask functions and it has been deleted in the past (!).